data/Grigg_2008/metadata.yml

```diff
@@ -0,0 +1,547 @@
+source:
+  primary:
+    key: Grigg_2008
+    bibtype: Article
+    year: '2008'
+    author: Alasdair M. Grigg and Erik J. Veneklaas and Hans Lambers
+    journal: Australian Journal of Botany
+    title: Water relations and mineral nutrition of closely related woody plant species
+      on desert dunes and interdunes
+    volume: '56'
+    number: '1'
+    pages: 27--43
+    doi: 10.1071/bt06205
+contributors:
+  data_collectors:
+  - last_name: Grigg
+    given_name: Alasdair
+    ORCID: unknown
+    affiliation: Plant Biology, Faculty of Natural and Agricultural Sciences, The
+      University of Western Australia
+  austraits_curators: Gillian Kowalick
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      group_by(Species) %>%
+        mutate(across(c(root_structure, plant_growth_form, nitrogen_fixing), replace_duplicates_with_NA)) %>%
```

Comment:
We don't want species-level traits read in multiple times; this functions fixes the problem.